libs/shared/src/types/jobs/index.ts

```diff
@@ -0,0 +1,12 @@
+import { EnvironmentId } from '../environment';
+import { OrganizationId } from '../organization';
+import { UserId } from '../user';
+
+export type JobId = string;
+
+export interface IJobData {
+  _id: JobId;
+  _environmentId: EnvironmentId;
+  _organizationId: OrganizationId;
+  _userId: UserId;
+}
```

Comment:
Minimal information we need to store in BullMQ to be able to run the jobs. Unfortunately for certain dependencies with certain features we can't only store the `JobId`.